cwr/README.md

```diff
@@ -6,10 +6,9 @@ a GraphQL API for querying the index.
 
 ## Overview
 
-This module convert CWR transactions ,as appears in CWR formatted files, to META objects and index these with
-certain keys.
+This module converts CWR transactions, as appears in CWR formatted files, to META objects and index these with certain keys.
```

Comment:
> This module converts CWR transactions, as appears in CWR formatted files, to META objects and index these with certain keys.

This module converts CWR transactions, as they appear in CWR formatted files, to META objects and indexes them on certain keys.